packages/addons-website/scss/components/_website-side-nav.scss

```diff
@@ -0,0 +1,260 @@
+@import '../prefix';
+
+.#{$prefix}--side-nav--website--with-header-nav {
+    width: 0;
```

Comment:
Nitpick - Tab width (or tab character) inconsistency?

Reply:
Thanks for reviewing!